pyproject.toml

```diff
@@ -92,7 +92,9 @@ doc = [
     "sphinx-autobuild==2021.3.14",
     "sphinx-autodoc-typehints==1.24.0",
     "sphinx-copybutton==0.5.2",
+    "sphinx-design==0.5.0",
     "sphinx-gallery==0.14.0",
+    "sphinx-notfound-page==0.8.3",
```

Comment:
I did remove this on purpose because it was giving an error/warning, but it seems you are not having it.

Fine then. Double check configuration. See:
https://github.com/readthedocs/readthedocs.org/blob/b13a321ba29d6b8e9651c7f42c2d23585333600b/docs/conf.py#L213